### PR TITLE
Cosmetic changes to README.md default values

### PR DESCRIPTION
**Pull request by benedekh**
* Default values are updated in the _Configuration Options_ section of README.md according to the index.js.
* They are always put on the last line of the comment for the corresponding argument. Although they may be redundant this way, but at least are easier to be recognized / distinguished in the comment.

They may conflict with other open PRs at the time of writing though.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ### 3.0.2 (Next)
 
 * Your contribution here.
+* [#85](https://github.com/alexa-js/alexa-app-server/pull/85): Default values are updated in the _Configuration Options_ section of README.md. - [@benedekh](https://github.com/benedekh).
```
Comment (dblock) [inline CHANGELOG.md]: No need for a CHANGELOG entry for READMEs.

**Comment by dblock** [comment]
Thanks! Just needs to be rebased please.

File: CHANGELOG.md (hunk 1 above)
Comment (benedekh) [inline CHANGELOG.md]: Sorry, I did not know that.

**Comment by dblock** [comment]
Thanks!

**Pull request merged**